spec/index.html

```diff
@@ -5462,31 +5462,55 @@ <h5>RDFterm-equal</h5>
+                <li>Produces a type error if <code>term1</code> and <code>term2</code> are both literals having the
+              same <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>; this datatype IRI is <em>not</em> in the
+              set of <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>; and the
+              <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical forms</a> of the two literals are different from one another.</li>
+                <li>Returns FALSE otherwise.</li>
+            </ul>
+            <p>
+              <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms if any of the following is
               true:</p>
             <ul>
               <li>
-                <span class="name">term1</span> and <span class="name">term2</span> are equivalent
-                <span class="IRI type">IRIs</span> as defined in <a data-cite=
-                                                                    "RDF12-CONCEPTS#section-Graph-URIref">6.4 RDF URI References</a> of
+                <span class="name">term1</span> is an <span class="IRI type">IRI</span> and <span class="name">term2</span> is an <span class="IRI type">IRI</span>
+                such that these two <span class="IRI type">IRIs</span> are equal as per the notion of <a data-cite=
+                                                                    "RDF12-CONCEPTS#dfn-iri-equality">IRI equality</a> of
                 [[RDF12-CONCEPTS]].
               </li>
               <li>
-                <span class="name">term1</span> and <span class="name">term2</span> are equivalent
-                <span class="literal type">literals</span> as defined in <a data-cite=
-                                                                            "RDF12-CONCEPTS#section-Literal-Equality">6.5.1 Literal Equality</a> of
+                <span class="name">term1</span> is a <span class="literal type">literal</span> and
+                <span class="name">term2</span> is a <span class="literal type">literal</span> such that
+                these two <span class="literal type">literals</span> are equal as per the notion of <a data-cite=
+                                                                            "RDF12-CONCEPTS#dfn-term-equal">Literal term equality</a> of
                 [[RDF12-CONCEPTS]].
               </li>
+              <li>
+                <span class="name">term1</span> is a <span class="literal type">literal</span> and
+                <span class="name">term2</span> is a <span class="literal type">literal</span>
+                such that the <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>
+                of each of these two literals is in the set of
+                <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>
```

Comment:
Is this intentionally dropping `the datatype IRI of the two literals is the same`? It doesn't seem like they should differ between the two literals....
```suggestion
                of each of these two literals is the same; this 
                <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a> is in the set of
                <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>;
```

Reply:
Ah, got it. Anyone who has such a button (I don't) can mark this Resolved.

Reply:
Marking this as resolved.